VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -699,18 +701,25 @@ export abstract class WaitingRoomBaseDirective {
         if (message.status !== ParticipantStatus.InConsultation && isMe) {
             this.isAdminConsultation = false;
         }
+        if (message.status === ParticipantStatus.Disconnected && participant) {
+            participant.current_room = null;
+        }
     }
 
     handleEndpointStatusChange(message: EndpointStatusMessage) {
         if (!this.validateIsForConference(message.conferenceId)) {
             return;
         }
 
-        const index = this.hearing.getEndpoints().findIndex(x => x.id === message.endpointId);
-        if (index === -1) {
+        const endpoint = this.conference.endpoints.find(p => p.id === message.endpointId);
```

Comment:
is the component no longer using the hearing object anywhere? if it still is then you'll need to update the conference and hearing object

Reply:
ive changed but i think theyre linked by ref, conference belongs to hearing